pallets/pallet-did-lookup/src/lib.rs

```diff
@@ -107,8 +109,19 @@ pub mod pallet {
 
 		type RuntimeHoldReason: From<HoldReason>;
 
+		type Balance: Parameter
+			+ Member
+			+ AtLeast32BitUnsigned
+			+ Codec
+			+ Default
+			+ Copy
+			+ MaybeSerializeDeserialize
+			+ sp_std::fmt::Debug
+			+ MaxEncodedLen
+			+ TypeInfo
+			+ FixedPointOperand;
```

Comment:
Can you not simply use MutateHold::Balance? Do you need the additional trait bounds?

Reply:
I was able to bring it down to [this](https://github.com/KILTprotocol/kilt-node/pull/547/commits/f57c084f26949ca0d95dc76b5c97d89f65e88700). If I remove the trait bounds completely, everything becomes red.

Reply:
Ok maybe this works... https://github.com/KILTprotocol/kilt-node/pull/547/commits/609a41b18776d2fc0304823ce894bd783afaccd0